src/azure-cli/HISTORY.rst

```diff
@@ -5216,4 +5216,4 @@ azure-cli-vm
 0.1.0b11 (2016-12-12)
 +++++++++++++++++++++
 
-* Preview release.
+* Preview release.
```

Comment:
Why there is a change here? Could we not modify it?

Reply:
fixed